scripts/build/build.ps1

```diff
@@ -34,7 +34,18 @@ header "Compiling"
 _ "$RepoRoot\scripts\compile\compile.ps1" @("$Configuration")
 
 header "Running Tests"
-_ "$RepoRoot\scripts\test\runtests.ps1"
+# Capture PATH for later
```

Comment:
Well, since `build.cmd` launches a powershell process, and there is no `build.ps1` in the root, we actually don't need to do this at all, since the build script always runs in it's own process.